vars/shift_stack_vars.yaml

```diff
@@ -6,7 +6,8 @@ osp_user_name: shiftstack_user
 # osp_project_name: "shiftstack{{ocp_cluster_index}}"
 # osp_user_name: "shiftstack{{ocp_cluster_index}}"
 
-ocp_release: 4.6.1
+ocp_release: 4.7.0-fc.5
+ocp_dev_preview: true
```

Comment:
Will this be used by shift-on-stack CI? If so it will be good if we keep
ocp_dev_preview: false
and
ocp_release to 4.6.1 which might be stable version.

Reply:
Got it, set it to those defaults.